tracecat/expressions/ioc_extractors/mac.py

```diff
@@ -0,0 +1,37 @@
+import re
+
+from pydantic import BaseModel, ValidationError, field_validator
+from pydantic_extra_types.mac_address import MacAddress
+
+# MAC ADDRESS
+MAC_REGEX = r"(?<![\d:A-Fa-f-])(?:[0-9A-Fa-f]{2}(?::[0-9A-Fa-f]{2}){5}|[0-9A-Fa-f]{2}(?:-[0-9A-Fa-f]{2}){5})(?![\d:A-Fa-f-])"
+
+
+class MacAddressModel(BaseModel):
+    mac_address: MacAddress
+
+    @field_validator("mac_address")
+    def normalize_format(cls, v):
```

Comment:
should specify whether this is a `before` `after` or `wrap` validator and add type hints for `v`. https://docs.pydantic.dev/latest/concepts/validators/#field-validators